apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+  .custom-colors[data-color-mode='dark'] {
+
+    /*
+     * Modify the value of these tokens.
+     * Remember to apply light mode equivalents if you're enabling theme switching.
+     */
+    --brand-CTABanner-shadow-color-start: var(--base-color-scale-purple-5-hsl);
+    --brand-CTABanner-shadow-color-end: var(--base-color-scale-red-5-hsl);
+
+  }
+`
+  return (
+    <ThemeProvider colorMode="dark" className="custom-colors">
+      <style>{designTokenOverrides}</style>
+      <CTABanner>
+        <CTABanner.Heading>
+          Where the most ambitious teams build great things
+        </CTABanner.Heading>
+        <CTABanner.Description>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+          ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+          elementum.
+        </CTABanner.Description>
+        <ButtonGroup buttonSize="large">
+          <Button>Primary Action</Button>
+          <Button>Secondary Action</Button>
+        </ButtonGroup>
+      </CTABanner>
+    </ThemeProvider>
+  )
+}
+
+render(<App />)
+```
+
+## Component props
+
+### ComparisonTable <Label>Required</Label>
+
+The first row in a table is automatically styled as a table header row.
+
+| name            | type                       | default   | required | description                                                  |
+| --------------- | -------------------------- | --------- | -------- | ------------------------------------------------------------ |
+| `children`      | `ReactNode`, `ReactNode[]` | undefined | `true`   | Content to include within the banner component               |
+| `align`         | `start`, `center`          | `start`   | `true`   | The layout applied to the content wihin the banner component |
+| `hasBorder`     | `boolean`                  | `false`   | `false`  | A flag used to provide a border to the banner component      |
+| `hasBackground` | `boolean`                  | `true`    | `false`  | A flag used to add a background to the banner component      |
+| `className`     | `string`                   |           | `false`  | CTABanner custom class                                       |
+
+### ComparisonTable.Heading
+
+Optional node that can be used to provide a heading for the banner.
+
+| name        | type                       | default   | required | description                                                           |
+| ----------- | -------------------------- | --------- | -------- | --------------------------------------------------------------------- |
+| `className` | `string`                   |           | `false`  | Custom class name for the ComparisonTable.Heading component           |
+| `children`  | `ReactNode`, `ReactNode[]` | undefined | `true`   | Content to be displayed inside the ComparisonTable.Heading component. |
+| `as`        | `h1`, `h2`                 | `h1`      | `false`  | Content to be displayed inside the column cell.                       |
```

Comment:
related feedback https://github.com/primer/brand/pull/128/files#r1035940710

Reply:
Other than the default which we will discuss in maintainers sync I updated it to include all options.